checkbox.js

```diff
@@ -0,0 +1,125 @@
+/*jshint esversion: 6*/
+/*global document*/
+/*global alert*/
+
+class Data {
+    static getCheckboxList() {
+        return ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday', 'None'];
+      } 
+  }
+  
+  class CheckboxOperations {
+    
+    createCheckboxList() {
+      let ul = this.createUlElementForCheckbox();
+      document.body.appendChild(ul);
```

Comment:
This should happen only when all elements are ready. First append all heckboxes to ul. Then append that ul to body.